cmd/parca-agent/main.go

```diff
@@ -153,6 +154,10 @@ func main() {
 	intro := figure.NewColorFigure("Parca Agent ", "roman", "yellow", true)
 	intro.Print()
 
+	if flags.ExperimentalEnableDWARFUnwinding && flags.MemlockRlimit != 0 && flags.MemlockRlimit < defaultMemlockRLimitWithDWARFUnwinding {
+		flags.MemlockRlimit = defaultMemlockRLimitWithDWARFUnwinding
```

Comment:
We can add a log message.

Also, is this still a sane value as the minimum required locked memory? @javierhonduco

Reply:
cc @Sylfrena as per our conversation today :)

Reply:
> The best way to check would be to sum up the reported locked memory. We can use `sudo bpftool map` to find the actual value for both when DWARF unwinding is enabled and disabled

Do we need to do this as part of boot strapping the agent or can we do it at development and specify a constant value?

Reply:
We can set it constant, it should not change right now (except of course for the dynamic setting of the unwind table max size! 😄 ) but we can add both manually perhaps with a small extra buffer, like 5% more memory or something in case the overhead of the kernel data structure changes or something, even though this would be quite unlikely I think!

Reply:
Ok, I'll run it and set the value.

Reply:
Should we round 402501632 bytes to 450MB?

Reply:
Sanity check, please: `402 MB` https://go.dev/play/p/-3UYhMpAep3

Reply:
> Should we round 402501632 bytes to 450MB?

Let's go with 512MB. I like it round :D

Reply:
`402 MB` looks right! Let's tighten this value for next release 😄